service/src/main/java/uk/gov/hmcts/reform/fpl/service/OthersService.java

```diff
@@ -47,90 +38,4 @@ private String getName(Other other) {
     private boolean otherExists(Others others) {
```

Comment:
for tech debt: if we also put buildOthersLabel into the helper class then this service can be removed as well (but since others is a bit more complicated maybe it's fine keeping it separated. Also nothing to do with your PR but just above this I see a "BLANK - Please complete" and that worries me 👀)

Reply:
https://github.com/hmcts/fpl-ccd-configuration/compare/FPLA-1175-confidential-tab-content...dev/generic-label-time 🙊 🙈